internal/pkg/githubapi/github.go

```diff
@@ -112,26 +115,18 @@ func HandlePREvent(eventPayload *github.PullRequestEvent, ghPrClientDetails GhPr
 			ghPrClientDetails.PrLogger.Errorf("Failed to minimize stale comments: err=%s\n", err)
 		}
 		if config.CommentArgocdDiffonPR {
-			var componentPathList []string
-
-			// Promotion PR have the list of paths to promote in the PR metadata
-			// For non promotion PR, we will generate the list of changed components based the PR changed files and the telefonistka configuration(sourcePath)
-			if DoesPrHasLabel(*eventPayload, "promotion") {
```

Comment:
yes, this logic was moved inside `generateListOfChangedComponentPaths`
And instead of checking the PR label I just check the Telefonistka PR metadata: 
```
// If the PR has a list of promoted paths in the PR Telefonistika metadata(=is a promotion PR), we use that
	if len(ghPrClientDetails.PrMetadata.PromotedPaths) > 0 {
```